src/app-routing.module.ts

```diff
@@ -3,18 +3,22 @@ import { RouterModule, Routes } from '@angular/router';
 import { DashboardComponent } from './components/dashboard/dashboard.component';
 import { VRScatterPlotComponent } from './components/vr-scatter-plot/vr-scatter-plot.component';
 import {VRScatterPlotModule} from './components/vr-scatter-plot/vr-scatter-plot.module';
+import { VRAccessibilityComponent } from './components/vr-accessibility/vr-accessibility.component';
+import { VRAccessibilityModule } from './components/vr-accessibility/vr-accessibility.module';
 import { DashboardModule } from './components/dashboard/dashboard.module';
 
 const routes: Routes = [
   { path: '', component: DashboardComponent, pathMatch: 'full' },
   { path: 'webVR', component: VRScatterPlotComponent, pathMatch: 'full' },
+  { path: 'accessibilityVR', component: VRAccessibilityComponent, pathMatch: 'full' },
 ];
 
 @NgModule({
   imports: [
     RouterModule.forRoot(routes),
     DashboardModule,
-    VRScatterPlotModule
+    VRScatterPlotModule,
+    VRAccessibilityModule
```

Comment:
World's biggest nit: it's slightly nicer to put a trailing comma here so that in the next diff, the only line that's changed is the one that is added.